src/native/fs/fs_napi.cpp

```diff
@@ -559,6 +559,11 @@ struct FSWorker : public Napi::AsyncWorker
             _report_fs_stats.Call({ fs_worker_stats });
         }
     }
+    bool use_gpfs_lib()
+    {
+        const char* gpfs_dl_path = std::getenv("GPFS_DL_PATH");
```

Comment:
we have this check in the initialization function on the bottom of the file, so how about moving the gpfs_dl_path variable to a static module like the dlsym_gpfs_fcntl and other symbols we load from the library? -

https://github.com/noobaa/noobaa-core/blob/590ce634e5cf0f3c9066e5064202c90d1bb82e5c/src/native/fs/fs_napi.cpp#LL1881C60-L1881C60

another option is to just check if one of those symbols is non zero...

Reply:
I moved it to be a static field, is that fine?